### DISCUSSION TITLE
Opening up the repo to the public

### DISCUSSION THREAD

**Discussion by erikbra** [grate-devs/grate#47] [General]
@wokket (and others, of course), do you see any issues with just making this repo public now? We need to make a first release soon, I thinkg, but it doesn't hurt opening up the source code before we make a first binary release, does it?

**Comment by wokket** [comment]
No issues with opening up from my side, other than the influx of requests/competing opinions we'll get...  We need those, but we also still have a fair bit to get done.  If you're happy managing the MVP scope (or publishing pre-release packages)...

I actually pinged an old work colleague that I started using RH with in 2013(? ish) with the set of outstanding features to see if they use any of them as an example 'legacy' userbase.

On a personal note my availability is going to drop for a few days. I should be able to push through some more next week, but then away for a week or so.

**Reply by erikbra** [reply]
Thanks for your comments. I fully understand not being able to work as much all the time. You have contributed a LOT the last couple of weeks, and  I really appreciate it. Don't feel you're obliged to contribute as much all the time! Any contributions are more than welcome!

**Reply by wokket** [reply]
I'm comfy contributing as much as I am at the moment while it's easy changes and needed to get something out.  It's really close to having all I need in  RH replacement, so well worth the effort.

**Comment by erikbra** [comment]
And we're open to the public :)
